packages/charts/react-charting/src/components/CommonComponents/CartesianChart.base.tsx

```diff
@@ -346,7 +357,12 @@ export class CartesianChartBase
           ));
           break;
         default:
-          ({ xScale, tickValues } = createNumericXAxis(XAxisParams, this.props.chartType, culture));
+          ({ xScale, tickValues } = createNumericXAxis(
+            XAxisParams,
+            this.props.tickParams!,
```

Comment:
>tickParams [](http://example.com/codeflow?start=23&length=10)

why do we need tickParams for numeric axis

Reply:
I added it just to make the functionality consistent across the 3 utilities. It will also allow users to provide custom tick values if they don't want to use the ones generated by D3.